--- lspconfig.lua	2024-02-02 20:23:40.269773073 +0100
+++ lspconfig.tmp	2024-02-02 21:47:41.205242495 +0100
@@ -8,7 +8,9 @@
 
 M.on_attach = function(client, bufnr)
   utils.load_mappings("lspconfig", { buffer = bufnr })
+end
 
+M.on_init = function(client, initialization_result)
   if client.server_capabilities.signatureHelpProvider then
     require("nvchad.signature").setup(client)
   end
@@ -39,6 +41,7 @@
 }
 
 require("lspconfig").lua_ls.setup {
+  on_init = M.on_init,
   on_attach = M.on_attach,
   capabilities = M.capabilities,
 
@@ -62,3 +65,4 @@
 }
 
 return M
+

